changes.txt

```diff
@@ -1 +1,13 @@
 /user/create.sql
+/transaction_type/create.sql
+/stock/create.sql
+/portfolio/create.sql
+/portfolio_balance/create.sql
+/portfolio_stock/create.sql
+/account_balance/create.sql
+/portfolio_member/create.sql
+/portfolio_follower/create.sql
+/portfolio_balance_transaction/create.sql
+/portfolio_stock_transaction/create.sql
+/account_balance_transaction/create.sql
+/stock_daily_price/create.sql
```

Comment:
It looks like this change set isn't being run. Each line needs to end with a newline character in order for it to be picked up by the changeset script.